pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -357,6 +357,48 @@ Upgrading catalog applications using Spark Operator can fail when running `dkp u
 
 When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, we recommend that you disable the `fluent-bit` Platform Application before upgrading. 
 
+### Calico not updated during DKP upgrade
+
+When upgrading a DKP cluster, after what seems to be a successful upgrade, the Calico service might not update as expected and, therefor, is still using the old image. The wrong CNI ClusterResourceSet is being generated and not accounting for Flatcar. This issue only impacts Calico, no other add-ons.
+
+Follow these steps to manually correct this issue:
+
+1.  Update the ConfigMap as follows:
+
+    ```
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    data:
+      custom-resources.yaml: |+
+        # This section includes base Calico installation configuration.
+        # For more information, see: https://docs.projectcalico.org/reference/installation/api
+        apiVersion: operator.tigera.io/v1
+        kind: Installation
+        metadata:
+          name: default
+        spec:
+          # Configures Calico networking.
+          calicoNetwork:
+            # Note: The ipPools section cannot be modified post-install.
+            ipPools:
+            - blockSize: 26
+              cidr: 192.168.0.0/16
+              encapsulation: IPIP
+              natOutgoing: Enabled
+              nodeSelector: all()
+            bgp: Enabled
+            nodeAddressAutodetectionV4:
+              firstFound: true
+          # FlexVolume path must be mounted under /opt on flatcar/coreos systems
+          flexVolumePath: /opt/libexec/kubernetes/kubelet-plugins/volume/exec/
+    kind: ConfigMap
+    metadata:
+      name: calico-cni-installation-c3-0193d
+    EOF
+    ```
+
+1.  Execute these commands: `kubectl edit calico-cni-installation-c3-0193d` and update `spec.clusterSelector.matchLabels.konvoy.d2iq.io/osHint` to `konvoy.d2iq.io/osHint: flatcar`
```

Comment:
@dkoshkin  - isn't this command missing the resource?  like 'kubectl edit deployment ...'???